sns_ik_examples/README.md

```diff
@@ -0,0 +1,41 @@
+# README:  sns_ik_examples
+
+This directory contains scripts that run some tests of the SNS-IK library using
+the Sawyer and Baxter robot models. There are three scripts:
+- `test_baxter.launch` compares the various IK solvers using the Baxter robot model
+- `test_sawyer.launch` compares the various IK solvers using the Sawyer robot model
+- `test_ik_solvers.launch` compares the various IK solvers on large random data sets
+
+## Tutorial: how to run these scripts
+
+First you need to setup your catkin workspace. If you are unfamiliar with this
+process then you should refer to the ROS tutorial page:
+http://wiki.ros.org/catkin/Tutorials/create_a_workspace
+
+Once you set up the catkin workspace it should look something like:
+````
+ros_ws/
+- src/
+  - sns_ik/
+  - sawyer_robot/
+````
+
+The `sns_ik/` directory contains this repository, cloned from:
+https://github.com/RethinkRobotics-opensource/sns_ik.
+The `sawyer_robot/` directory contains the open-source model of the Sawyer robot, cloned from:
+https://github.com/RethinkRobotics/sawyer_robot.
```

Comment:
To run the baxter test, they will need baxter_common:
https://github.com/RethinkRobotics/baxter_common.git

Reply:
good idea - I added that to the README